Dockerfile.test-infra

```diff
@@ -22,5 +22,7 @@ RUN pip3 install --no-cache-dir -r /tmp/requirements.txt && \
 # TODO: Remove once OpenShift CI will be upgraded to 4.2 (see https://access.redhat.com/articles/4859371)
 RUN chmod uga+w /etc/passwd
 
+RUN curl -Lo oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/oc/${OPENSHIFT_VERSION:-4.5}/linux/oc.tar.gz && tar -C /usr/local/bin -xf oc.tar.gz && rm -f oc.tar.gz
```

Comment:
I need explanation why we need oc

Reply:
in order to get remote assisted-service url.
only for the login command inside skipper.

Reply:
looks like it has python package, sounds like better idea to use it